src/TabNavigator.re

```diff
@@ -322,6 +331,10 @@ module CreateTabNavigator = (Config: {type route;}) => {
           screens[index] = {...screens[index], tabItem};
           ReasonReact.Update({...state, screens});
         },
+      didMount: self => {
+        onNavigationReady(getNavigationInterface(~send=self.send, ~screens=self.state.screens, ~currentRoute=self.state.currentRoute, ~isActive=true, ~index=0));
```

Comment:
needed for deeplinking
we are using this so we can lift to a parent the state handling and navigation actions